apps/mobile/src/core/apis/provider.ts

```diff
@@ -313,6 +323,7 @@ type gasMarketV2ParamsV1 = {
 };
 export const gasMarketV2 = async (
   _params: gasMarketV2ParamsV1 | gasMarketV2ParamsV2,
+  account: Account,
```

Comment:
So we also need update usage in `apps/mobile/src/screens/Send/hooks/useSendToken.ts#L331`, please check it.